controller/index.js

```diff
@@ -29,10 +29,24 @@ var util = require('util'),
 
 var Generator = module.exports = function Generator(args, options, config) {
     yeoman.generators.Base.apply(this, arguments);
+    var cwd = this.env.cwd,
+        filePath = path.join(cwd, 'package.json'),
+        packageTemplate = null;
 
     krakenutil.update();
 
-    this.hasTemplates = (args[1]) ? true : false;
+    this.hasTemplates = (args[1]) ? true : (
+        fs.existsSync(filePath) ? (
```

Comment:
looks like you have forgotten to require `fs` causing the build to break for your pr. Can you fix it ?